Edit Datasets documentation page to include link to noise page and other small changes (#106)
ihmeuw/pseudopeople: docs/source/datasets/index.rst

Datasets
========

Here we cover the realistic simulated datasets, which are analogous to "real world" administrative records such as tax documents

> and census surveys, that users can generate using Pseudopeople for developing and testing Entity Resolution algorithms 

Reviewer: ```suggestion
and routinely generated files of social security numbers, that users can generate using Pseudopeople for developing and testing Entity Resolution algorithms 
```
and did we make a decision about `pseudopeople` vs pseudopeople vs Pseudopeople?  I think we are headed for lowercase and not fixed width.

and software. 

> Each of the datasets that can be generated using Pseudopeople can have "noise" added to them, thereby realistically 
> simulating how administrative records can be corrupted or distorted, adding extra challenges to researchers trying to link those 

Reviewer 2: ```suggestion
Each of the datasets that can be generated using Pseudopeople have "noise" added to them, thereby realistically 
simulating how administrative records can be corrupted or distorted, which creates challenges in linking those 
```

that have been simulated are 2020, 2030, and 2040. To find out more about the Decennial Census, please visit the Decennial Census
`homepage <https://www.census.gov/programs-surveys/decennial-census.html>`_.   

> To find out more about how to generate a simulation of the Decennial Census using Pseudopeople, see :func:`pseudopeople.interface.generate_decennial_census`.

Reviewer: @albrja did you just say you have a way to link directly to `psp.generate_decennial_census`?

that have been simulated are 2020, 2030, and 2040. To find out more about the Decennial Census, please visit the Decennial Census
`homepage <https://www.census.gov/programs-surveys/decennial-census.html>`_.   

> To find out more about how to generate a simulation of the Decennial Census using Pseudopeople, please see the `API Reference page <https://pseudopeople.readthedocs.io/en/latest/api_reference/interface.html#pseudopeople.interface.generate_decennial_census>`_ for the 
> :code:`generate_decennial_census()` function. 

Reviewer 2: ```suggestion
To find out more about how to generate a simulation of the Decennial Census using Pseudopeople, see :func:`pseudopeople.interface.generate_decennial_census`.
```

I think we should make this short, and repeat it for every dataset. I think we need to have the "interface" part of this link for now but I'm going to follow up with the engineers about it on Slack.
